ios/Discourse.xcodeproj/xcshareddata/xcschemes/Discourse-tvOS.xcscheme

```diff
@@ -1,88 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Does this mean Discourse is no longer available on tvOS? 😅 (I don't know if it really was before 😛)

Reply:
Naah, we never tested/released the app for tvOS.